### PR TITLE
Add recommendation for Windows 7

### PR DESCRIPTION
**Pull request by pamolloy**
See https://discuss.luxonis.com/d/105-run-on-win7-sp1-x64-manual-instal-usb-driver

### CODE REVIEW

File: source/pages/api.rst [hunk 1]
```diff
@@ -92,9 +93,26 @@ Windows
...
+###############
+
+PyPI
+Docker
+Podman
+VirtualBox
+QEMU
+apt
+rpm
+C++
```
Comment (VanDavv) [inline source/pages/api.rst:115]: why are these listed here?

File: source/pages/api.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@ Python API
 
 Please :ref:`install the necessary dependencies <Supported Platforms>` for your
 platform by referring to the table below. Once installed you can :ref:`install
-the DepthAI library <Install from PyPi>`.
+the DepthAI library <Install DepthAI>`.
```
Comment (VanDavv) [inline source/pages/api.rst:6]: The value inside `<...>` is a header reference, so `Install from PyPi` is an actual header

**Review by VanDavv — CHANGES_REQUESTED** [review]

File: source/pages/api.rst (hunk 1 above)
Comment (pamolloy) [reply source/pages/api.rst]: Ah, glad you saw that! They're just notes for more edits I have planned. I'll dump them into a separate branch so I don't add them by accident again.

File: source/pages/api.rst (hunk 2 above)
Comment (pamolloy) [reply source/pages/api.rst]: I had renamed the header to `Install DepthAI` along with the notes mentioned in the other comment.

**Review by VanDavv — APPROVED** [review]

**Pull request merged**